app/assets/javascripts/application.js

```diff
@@ -74,67 +57,25 @@ $(document).on('turbolinks:load', function() {
     scope.find('[data-tab="' + target + '"]').addClass('active');
   });
 
-  $(".stick_ip").stick_in_parent()
-    .on("sticky_kit:stick", function(e) {
-      if($('.bread_links span').length)
-        return;
-      var title = $('h1').text();
-      var breadLinksHtml = $('.bread_links').html();
-      $('.bread_links').html(breadLinksHtml + ' <span>' + title + '</span>');
-    })
-    .on("sticky_kit:unstick", function(e) {
-      var sep = ' » ';
-      var title = $('h1').text();
-      var breadLinksHtml = $('.bread_links').html();
-      var arr = breadLinksHtml.split(sep);
-      arr.pop();
-      $('.bread_links').html(arr.join(sep) + sep);
-    });
-
-  $('.bread_hover').hover(function(e) {
-    $('.line_browser').velocity("fadeIn", { duration: 50 });
-  }, function(e) {
-    $('.line_browser').velocity("fadeOut", { duration: 50 });
-  });
-
-  $('.open_line_browser').click(function(e) {
-    e.preventDefault();
-    e.stopPropagation();
-    $('.line_browser').velocity("fadeIn", { duration: 250 });
-  });
-
-  $('.close_line_browser').click(function(e) {
-    e.preventDefault();
-    e.stopPropagation();
-    $('.line_browser').velocity("fadeOut", { duration: 150 });
-  });
-
-  $(window).click(function(){
-    $('.line_browser').velocity("fadeOut", { duration: 150 });
-  });
-
+  // Modal windows
   $('.open_modal').magnificPopup({
     type: 'inline',
     removalDelay: 300,
     mainClass: 'mfp-fade'
   });
 
-  $('.close_modal').click(function(e) {
-    $.magnificPopup.close();
+  $('.open_remote_modal').magnificPopup({
+    type: 'ajax',
+    removalDelay: 300,
+    mainClass: 'mfp-fade'
   });
 
-  var $autocomplete = $('[data-autocomplete]');
-
-  var searchOptions = {
-    serviceUrl: $autocomplete.data('autocomplete'),
-    onSelect: function(suggestion) {
-      Turbolinks.visit(suggestion.data.url);
-    },
-    groupBy: 'category'
-  };
 
-  $autocomplete.autocomplete($.extend({}, AUTOCOMPLETE_DEFAULTS, searchOptions));
+  $('.close_modal').click(function(e) {
+    $.magnificPopup.close();
+  });
 
+  // TODO: move to gobierto budget consultations
   $('.carousel').slick({
```

Comment:
Yeah, it's not implemented yet, but it's used in the sandbox template. So I'll wait until we finish the budget consultations module to remove it or move it to another part.